packages/app/src/vis-packs/core/matrix/MatrixToolbar.tsx

```diff
@@ -16,8 +16,8 @@ import { useMatrixConfig } from './config';
 const EXPORT_FORMATS: ExportFormat[] = ['npy', 'csv'];
 
 interface Props {
-  dataset: Dataset<ArrayShape, PrintableType>;
-  selection: string | undefined;
+  dataset: Dataset<ArrayShape>;
+  selection?: string | undefined;
```

Comment:
Allow `selection` to be omitted in `MappedCompoundMatrixVis`. Otherwise, I have to set it to `undefined` explicitely :shrug: